Octokit.Tests/Reactive/ObservablePullRequestsClientTests.cs

```diff
@@ -225,7 +225,7 @@ public class TheMergeMethod
             [Fact]
             public void MergesPullRequest()
             {
-                var mergePullRequest = new MergePullRequest { CommitMessage = "fake commit message" };
+                var mergePullRequest = new MergePullRequest { CommitMessage = "fake commit message", CommitTitle = "fake title" };
```

Comment:
It looks like we're not asserting anything about this object, so I think we should revert this change. :cool:?